pytest.ini

```diff
@@ -1,6 +1,6 @@
 [pytest]
 log_cli = 1
-log_cli_level = DEBUG
+log_cli_level = CRITICAL
```

Comment:
this one reduce live output  of python logging by pytest
logging output will be hidden till test fails